pyanaconda/core/configuration/payload.py

```diff
@@ -61,6 +61,17 @@ def enable_closest_mirror(self):
         """
         return self._get_option("enable_closest_mirror", bool)
 
+    @property
+    def enable_addon_repos(self):
+        """Enable installation from .treeinfo addon repos.
+
+        This flag controls whether or not Anaconda will automatically
+        enable any addon repos defined in .treeinfo for your product.
```

Comment:
Please, mention that the base repository is always enabled.

Reply:
I'll make this change shortly.